### DISCUSSION TITLE
How can I remove accessibility outline from switch component

### DISCUSSION THREAD

**Discussion by mingbackmountain** [chakra-ui/chakra-ui#3647] [Q&A]
Hi guys I have tried to remove the blue outline around the switch component 

<img width="54" alt="Screen Shot 2564-03-23 at 16 45 06" src="https://user-images.githubusercontent.com/25886138/112126619-35c1ac00-8bf7-11eb-8140-c0f436a16712.png">
<img width="61" alt="Screen Shot 2564-03-23 at 16 45 11" src="https://user-images.githubusercontent.com/25886138/112126624-36f2d900-8bf7-11eb-9e92-0cebc60d6542.png">

I have tried to use the _focus props like this

`<Switch colorScheme="blue" _focusVisible={{outline:"none"}} _focus={{outline:"none"}}/>`

But it did not work, I have used a similar prop with Box and Button and it worked

**Comment by malashevskyi** [comment]
Hi.
In this way:
```javascript
const theme = extendTheme({
  components: {
    Switch: {
      baseStyle: {
        track: {
          _focus: {
            boxShadow: "none"
          }
        }
      }
    }
  }
})
```

**Reply by mingbackmountain** [reply]
Thank you so much
